docs/sdk/0-core-feature/1_web3names.md

```diff
@@ -63,4 +69,46 @@ The KILT SDK provides easy-to-use querying functions to perform both operations
 
 <CodeBlock className="language-js">
   {Example4}
+</CodeBlock>
+
+### Linking accounts to DIDs and Web3 names
+
+Beyond linking a Web3 name, KILT allows DID owners to link multiple accounts under the same DID.
+These accounts are not specific to the KILT blockchain but can reference any chain within the Kusama ecosystem.
+Each account <-> DID link requires the payment of a small deposit, which is returned whenever the link is removed.
+
+For DIDs that have also claimed a Web3 name, the linking feature opens up the way to a host of possibilities, e.g., showing the Web3 name of a collator's account on the [KILT stakeboard](https://stakeboard.kilt.io/).
+
+An account can be linked to a DID in one of two ways:
+
+1. The account to be linked generates a signature over the raw byte-encoded `(SubmitterDid, BlockNumberExpiration)` payload:
```

Comment:
well this has changed now hasn't it

Reply:
Yeah that's why I wrote raw byte-encoded. Do you think we should make it explicit?